docs/ossm/ambient/waypoint.md

```diff
@@ -0,0 +1,233 @@
+# Getting Started with Istio Waypoint in Ambient Mode (Tech Preview)
+
+This section provides a high-level overview and installation procedure for Istio's ambient mode on OpenShift Container Platform (OCP) using OpenShift Service Mesh 3.1.x.
```

Comment:
```suggestion
Once you have gotten started with Istio's ambient mode using ztunnel proxies, you may want to introduce waypoint proxies to take advantage of the many Layer 7 (L7) processing features that Istio provides. This section provides an overview and procedure for adding waypoint proxies to workloads running in your Istio ambient mode service mesh.
```
It looks like the intro was copied from the previous section, so trying to introduce this page more accurately.